src/vega.schema.ts

```diff
@@ -1,8 +1,8 @@
-import {isArray} from './util';
+import {isArray, isObject} from './util';
 import {ScaleType, NiceTime} from './scale';
 
 export interface VgData {
-  name: string;
+  name?: string;  // TODO: make required
```

Comment:
Because we don't need the name until we actually output vega. The issue is that the data component != VgData.